zephyr/CMakeLists.txt

```diff
@@ -712,6 +712,12 @@ zephyr_library_sources_ifdef(CONFIG_COMP_COPIER
 	${SOF_AUDIO_PATH}/copier.c
 )
 
+zephyr_library_sources_ifdef(CONFIG_MAXIM_DSM
+	${SOF_AUDIO_PATH}/smart_amp/smart_amp.c
+	${SOF_AUDIO_PATH}/smart_amp/smart_amp_generic.c
+	${SOF_AUDIO_PATH}/smart_amp/smart_amp_maxim_dsm.c
+)
+
```

Comment:
pretty sure this doesn't belong to this commit.

Reply:
@dbaluta In principle we could separate changes to wrapper.c and to CMakeLists.c for each component, but this is a very small and simple and standard piece of code. It's very similar to what is done for all other components. No magic there at all.

Reply:
Sure, maybe add some comments inside the commit message. Saying that we also enable the component for zephyr?

Reply:
lets make this another commit.